docs/_extensions/apilinks.py

```diff
@@ -13,6 +13,8 @@
 from types import MappingProxyType
 
 
+apilinks_base_url = "https://docs.twisted.org/en/stable/api/"
```

Comment:
Maybe use latest twisted docs for now

```suggestion
apilinks_base_url = "https://docs.twisted.org/en/latest/api/"
```

----------

I see `:api:` custom directive is used 38 times.
Maybe after twitsted 22.8.0 is relased, in a separate PR we (I) can migrate to intrsphinx links and remove the custom extension.

In twisted/twisted we no longer use the `:api:` custom directive and I have submitted a PR to remove that code.

Reply:
Seems to work now!